main/solution/post-deployment/config/saml-metadata/datalake-demo-idp-metadata.xml

```diff
@@ -0,0 +1,1044 @@
+<!--
+  This file needs to be uploaded to S3 first before running post deployment steps.
```

Comment:
Can you add a comment of what this file is and what it is used for?

Reply:
This is an accidental file. We need to remove `main/solution/post-deployment/config/saml-metadata/datalake-demo-idp-metadata.xml` file before merging this PR. This file contains SAML metadata for a test ADFS instance we used.

Reply:
Done

Reply:
Yes, you are right.  Thanks for spotting this.